src/main/java/mods/railcraft/common/worldgen/GeneratorMine.java

```diff
@@ -38,7 +38,7 @@
  */
 public abstract class GeneratorMine extends Generator {
     private static final EnumSet<BiomeDictionary.Type> RICH_BIOMES = EnumSet.of(MOUNTAIN, MESA, HILLS);
-    private static final boolean SKY_GEN = false;
+    private static final boolean SKY_GEN = true;
```

Comment:
To all who decide to try this out: Note that this is currently enabled for the purposes of testing ore generation. This will not remain on when this is made ready to merge.